bidwire/scrapers/massgov/results_page_scraper.py

```diff
@@ -0,0 +1,26 @@
+from lxml import etree, html
+
+
+def scrape_results_page(page_str, xpath_list):
+    """Scrapes HTML page and returns dictionary of URL => document title
+
+    Args:
+    page_str -- the entire HTML page as a string
+    xpath_list -- list of xpath expressions (section and link element)
+
+    Returns:
+    document_ids -- a dictionary of relative URL path => description
+    """
+    document_ids = {}
+    if not xpath_list or len(xpath_list) < 2:
```

Comment:
Is this ever expected to happen? Wouldn't this be programmer error? If so, this probably should be an `assert` or some other exception-throwing code, and not just a check that returns an empty dictionary.